SRC/system_of_eqn/eigenSOE/ArpackSolver.h

```diff
@@ -46,7 +46,8 @@
 
 #include <EigenSolver.h>
 #include <ArpackSOE.h>
-#include <f2c.h>
+
+// #include <f2c.h>
 
```

Comment:
`f2c.h` moved from header to source file to avoid issues that `f2c.h` can cause in units that include `ArpackSolver.h`